chart/values.yaml

```diff
@@ -81,6 +81,10 @@ rancherImage: rancher/rancher
 # Defaults to .Chart.appVersion
 # rancherImageTag: v2.0.7
 
+# Override imagePullPolicy for rancher server images
+# Defaults to IfNotPresent
```

Comment:
Should we show the acceptable values? We do for tls and addLocal values.

Reply:
We could, but I figured it wasn't needed because this works on default functionality, where `tls` and `addLocal` use our own custom logic. I've added the list of the options.